index.html

```diff
@@ -224,21 +243,32 @@ <h4>Skills</h3>
         </section>
 
         <footer>
-            <div class="remarks">
-                <p>
-                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
-                </p>
+            <div class="last-part">
+                <div class="remarks">
+                    <p>
+                        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                    </p>
+                </div>
+    
+                <form method="POST" action="https://formspree.io/f/xzbqlowk" class="footer-form">
+                    <div class="full-name">
+                        <input type="text" name="first name" maxlength="30" placeholder="First name" class="first-name">
+                        <input type="text" name="last name" maxlength="30" placeholder="Last name" class="last-name">
+                    </div>
+                    <input type="text" name="full name" maxlength="30" placeholder="Full name" class="name" required>
+                    <input type="email" name="email" placeholder="Email address" class="email" required>
+                    <textarea name="comments" cols="30" rows="5" maxlength="500" placeholder="Hey I’d like to connect to..."></textarea>
+                    <input type="submit" value="Get in touch" class="get-button1">
+                    <p>wilsanfourrspeed@gmail.com</p>
+                </form>
```

Comment:
- [ ] As requested by the previous CR, please 🙏🏽 correct the items in the contact form section:

> The text on the left hand side is meant to be centered across the form element.
> The form input and textareas are supposed to be wider than that.
> The background images aren't properly fixed, please use the background css property position them properly using the css background-position property, and other background properties, you can make your research.
> The button Get in touch shouldn't be in the center.
> Add spacing between First name and Last name input tags

1 Figma design | Your design |
|---|---|
| ![image](https://github.com/Wils-FOURR-speed/My-portfolio/assets/58642949/4403b5c0-774f-4401-83f7-526254bf7f6a) | ![image](https://github.com/Wils-FOURR-speed/My-portfolio/assets/58642949/92dcc0b8-d29b-4ed6-b999-f72fe3a0a3fc) |